### PR TITLE
Add sponsor call to action

### PR DESCRIPTION
**Pull request by janz93**
* add first sponsor's
* add sponsor package information link

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -99,6 +114,22 @@ Read the full [Berlin Code of Conduct here](https://berlincodeofconduct.org/).
 
 </div>
 
+<div class="content-section content-section--whitebg sponsors" markdown="1">
+
+## Sponsors
+
+<div class="sponsors__listing sponsors__listing--gold" markdown="1">
+[![sum.cumo](assets/images/sponsors/sumcumo.svg)](https://www.sumcumo.com/)
```
Comment (emrox) [inline index.md]: why listing sum.cumo twice?

File: index.md [hunk 2]
```diff
@@ -35,8 +35,23 @@ With the purchase of a diversity ticket you help us to make this happen.
...
+<div class="content-section content-section--whitebg sponsors" markdown="1">
+
+## Sponsors
+### Gold
+
+<div class="sponsors__listing sponsors__listing--gold" markdown="1">
+[![sum.cumo](assets/images/sponsors/sumcumo.svg)](https://www.sumcumo.com/)
+</div>
+</div>
+<div class="content-section" markdown="1">
```
Comment (emrox) [inline index.md]: add empty line between sections

File: index.md (hunk 1 above)
Comment (janz93) [reply index.md]: Because the first one is for gold only and the last section are all current sponsors

Comment (skrings) [reply index.md]: @janz93 I am with @emrox here. One mention per Sponsor on the main page is enough from my point of view. Same procedure as last year. :)

Comment (janz93) [reply index.md]: Okay I guess now we have enough sponsors to seperate them again.
I'll update the PR

Comment (janz93) [reply index.md]: @skrings and @emrox I updated the sponsor section and divided them as we did it last year.
@skrings if you have any better wording for the community sponsor let me know.
My goal is the have this merged by Wednesday before the ruby on ice on Munich is started

**Comment by janz93** [comment]
Should be merged together with #14

**Comment by skrings** [comment]
@janz93 Thanks for starting this! Two things have struck me:

 * Last year the sponsor sections had different background colors than the previous and next section. I liked this solution more because of better readability.
 * It seems you somehow messed with the styles. Have a look at a purple section they are not 100% width anymore.

Please have a look at: https://2018.rubyunconf.eu

**Comment by skrings** [comment]
@janz93 Can you add XING as a sponsor as well please.

**Comment by janz93** [comment]
@skrings the xing and sum.cumo logo is not working well together as they have different `height` and `length`. For the first sponsoring information display for tomorrow, I would launch it without xing and add them a bit later.

**Comment by janz93** [comment]
@emrox  thanks for the coloring update :)

**Review by skrings — APPROVED** [review]
Resolve the conflict and then merge it! 🍰

**Pull request merged**